tools/convert-help-center-docs-to-mdx

```diff
@@ -148,6 +150,66 @@ def replace_icons(markdown_string: str, import_statement_set: set[str]) -> str:
     return result
 
 
+def convert_tab_syntax(markdown_string: str, import_statement_set: set[str]) -> str:
+    """
+    Convert our custom tab syntax to relevant MDX equivalent.
+    This function is inspired from `tabbed_section.py`'s run method.
+    """
+    TABBED_SECTION_TEMPLATE = """<Tabs>
+{blocks}
+</Tabs>"""
+
+    TAB_ITEM_TEMPLATE = """<TabItem label="{tab_label}">
+{content}
+</TabItem>"""
+
+    def indent_content_inside_blocks(content_block: str) -> str:
+        """
+        The original function `generate_content_blocks` does no
+        indenting, and we need the content in between our TabItems
+        to be indented. This is bit of string manipulation, but it
+        is a better alternative compared to duplicating the original
+        function here just to indent.
+        """
+        content_block_lines = content_block.splitlines()
+        result_lines = []
+        for line in content_block_lines:
+            if line.startswith(("<TabItem", "</TabItem>")):
+                result_lines.append(line)
+            else:
+                result_lines.append(INDENT_SPACES + line)
+
+        return "\n".join(result_lines)
+
+    lines = markdown_string.splitlines()
+    tab_section = parse_tabs(lines)
+    while tab_section:
+        if "tabs" in tab_section:
+            content_blocks = generate_content_blocks(tab_section, lines, TAB_ITEM_TEMPLATE)
+            content_blocks = indent_content_inside_blocks(content_blocks)
+            content_blocks = indent(content_blocks, INDENT_SPACES)
+            tabs_mdx = TABBED_SECTION_TEMPLATE.format(blocks=content_blocks)
+            import_statement_set.add(
+                "import { Tabs, TabItem } from '@astrojs/starlight/components'"
+            )
+        else:
+            # This is the case where we don't have any tabs but we were
+            # using the `start_tabs` and `end_tabs` syntax to create
+            # a border around the instructions. We just put the content
+            # as is in this case since we don't want to add that border
+            # anymore.
```

Comment:
I guess this is ultimately a question about styling -- if we use a different Astro theme for steps components, we might change our mind on this. Seems OK to merge this was and revisit that when we're doing final styling.